src/utils/buildProject.js

```diff
@@ -0,0 +1,54 @@
+import axios from 'axios';
+
+const MAX_CONTENT_LENGTH = 100 /* MB */ * 1024 * 1024;
+
+export default async function buildApkAsync(
+  appJson : Object,
+  headers : Object,
+  options : Object
+) {
+
+  const exp = appJson.expo;
+
+  return await _callMethodAsync('build', 'put', headers, {
```

Comment:
There's only the one method to call, so I don't think we need something like _callMethodAsync